.travis.yml

```
@@ -15,7 +15,7 @@ branches:
 
 stages:
   - name: test
-    if: NOT ((commit_message =~ /(skip-test-stage)/ AND branch = master) OR branch = /^(?i:feature).*$/ )
+    if: NOT (commit_message =~ /(skip-test-stage)/)
```

```suggestion
    if: NOT (commit_message =~ /(skip-test-stage)/ ) or branch = master
```
The test shoud ever run on master, i think